crowd_anki/config/config_dialog.py

```diff
@@ -32,6 +32,11 @@ def accept(self):
             super().accept()
 
     def ui_initial_setup(self):
+        self.ui_setup_snapshot_options()
```

Comment:
I don't think we need the `ui` in the name, we're in the ui widget after all ;).

Reply:
Yea sure why not :ok: